draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -627,6 +627,28 @@ in cache before performing any followup queries.
 With these optimizations in place, and conforming DNS servers,
 using SVCB does not add network latency to connection setup.
 
+Moreover, if an A or AAAA response arrives before an SVCB response, the
+client SHOULD wait up to CD milliseconds before connecting as if the
+SVCB query returned NODATA, but MUST NOT transmit any information that
+could be altered by an SVCB response until the SVCB response arrives.
+For example, a TLS connection may be altered by the "esnikeys" value
```

Comment:
I think "ClientHello" would be clearer than "connection"

Reply:
I didn't want to bubble up too much of TLS, but if you're fine with it, that works for me!